addons/web/static/src/core/confirmation_dialog/confirmation_dialog.xml

```diff
@@ -5,8 +5,8 @@
     <Dialog size="'md'" title="props.title" modalRef="modalRef">
       <p t-out="props.body" class="text-prewrap"/>
       <t t-set-slot="footer">
-        <button class="btn" t-att-class="props.confirmClass" t-on-click="_confirm" t-esc="props.confirmLabel"/>
-        <button t-if="props.cancel" class="btn btn-secondary" t-on-click="_cancel" t-esc="props.cancelLabel"/>
+        <button class="btn" t-att-class="props.confirmClass" t-on-click="_confirm" t-esc="props.confirmLabel" data-hotkey="q"/>
+        <button t-if="props.cancel" class="btn btn-secondary" t-on-click="_cancel" t-esc="props.cancelLabel" data-hotkey="x"/>
```

Comment:
I chose "Q" and "X" because these are the hotkeys on action view dialogs.
It could be "S" and "J" like other dialogs in the frontend.
It could be "C" and "J" like the `FormViewDialog`.

I don't have any opinion on that, except that it would be a good idea 1) to standardize all our dialogs instead of choosing different letters half the time, 2) to make sure hotkeys are used everywhere, else we can't count on them and they're useless. :shrug:

Reply:
This is fine to me, I don't have any strong opinion on this as, like you said, we don't really have any standardization for shortcuts.